introduction.md

```diff
@@ -12,4 +12,5 @@ The weapons I am profissioned with, are
 * c#
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?